tests/e2e/e2e_setup_test.go

```diff
@@ -33,6 +33,14 @@ type syncInfo struct {
 	SyncInfo status `json:"SyncInfo"`
 }
 
+type valinfo struct {
+	VotingPower string `json:"VotingPower"`
```

Comment:
json tag should be `voting_power`, no?

Reply:
This was originally being pulled from `osmosisd status` which shows as `VotingPower` even when queried as json. I actually no longer use this, so will remove it.